src/index.js

```diff
@@ -99,7 +102,9 @@ export default class PlayerComponent extends Component {
         }
         clearInterval(this.stepInterval);
         this.setState({
-            playerState: STATE.ENDED
+            playerState: STATE.ENDED,
+            progressValue: 100,
+            currentPos: duration
         });
```

Comment:
When playback ends,
1. sets audio progress to 100.
2. sets current position same as audio duration.